website_sale_custom_filter/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_website_filter_public,website.filter.public,model_website_sale_custom_filter,,1,0,0,0
+access_website_filter_values_public,website.filter.values.public,model_website_sale_custom_filter_value,,1,0,0,0
```

Comment:
do we need full access for group_website_designer?

Reply:
I'd say yes we need for that - as per module specification:
```
Only users from group website.group_website_designer should have CRUD rights on these models.
Public users must have READ access.
```